### PR TITLE
new project space

### PR DESCRIPTION
**Pull request by urgetolearn**
## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

**Comment by tkuhrt** [comment]
As a first step, please add DCO sign-off to this commit. Please see the instructions on the [checks tab](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/277/checks?check_run_id=28537773455)

File: proposal-template.md
Comment (tkuhrt) [inline proposal-template.md:1]: Please do not change this file. Please copy the template into the labs folder with as `name_of_lab.md`. Then make the changes specified.

File: proposal-template.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 # Short Description
-_Provide a short description of your lab. This will be used for the Github repository's description._
+A VS code extension and toolbox to debugging a chaincode. The extension once installed can be used by developers of chaincode to run and debug from their IDE, to understand how their chaincode behaves.
 
 # Scope of Lab
-_Enter the scope of the lab. This should include enough information to allow the Lab stewards to understand how it fits within Hyperledger's mission._
+Hyperledger Fabric chaincode developers often have to repeat the same steps to set up their environment and deploy code for debugging. This process can be tedious and time-consuming. If the IDE had a built-in debugging interface, it could automate these tasks, making the developer's job easier and faster.
 
 # Initial Committers
 _Enter the Github IDs for the set of initial committers._
```
Comment (tkuhrt) [inline proposal-template.md:17]: After you copy the template, you will need to fill in all details of the template.

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Please make requested changes

**Comment by urgetolearn** [comment]
This PR was accidentally created, I have now created a new PR for the project proposal.

**Pull request closed without merging**